webpack.config.build.js

```diff
@@ -0,0 +1,35 @@
+const webpackCommon = require('./webpack.config.common');
+const merge = require('webpack-merge');
+const webpack = require('webpack');
+const path = require('path');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const nodeExternals = require('webpack-node-externals');
+
+const config = merge(webpackCommon, {
+  devtool: 'source-map',
+  entry: {
+    main: './src/index.js'
+  },
+  plugins: [
+    new ExtractTextPlugin('styles.css'),
+  ],
+  target: 'node',
```

Comment:
You probably know this better than me, but don't we want to target `web` rather than `node` here?

Reply:
No, TBH I copied over the defaults form `metabolica-ui-map`, which probably inherited it from a boilerplate project. The output of this build goes to node_modules. I'll read up on this tomorrow

Reply:
So my research didn't result in anything reliable, this is what I found: https://gist.github.com/mxstbr/7723a02e1156d35f04802870a5355694#file-webpack-for-node-modules-js-L27

Reply:
Have you looked at [the docs](https://webpack.js.org/configuration/target/)? My understanding is that since we're packaging for use in a browser, and not server environment with node.js, we want to target `web`.